README.md

```diff
@@ -5,6 +5,105 @@ on top of [Celery](https://docs.celeryproject.org/).
 * Celery tasks interface
 * Anti-flood tactics
 
+Installation
+------------
+
+```shell script
+pip install celery-amqp-events
+```
+
+Configuration
+-------------
+
+1. Set a "service name" for each service that has event handlers.
+2. Tell celery with `imports` settings where to find event handlers.
+3. Configure broker connection and other celery settings
+4. Leave result backend empty - each event may have multiple consumers,
+    event result is meaningless in this case.
+> You absolutely need to set separate name for each service that consumes 
```

Comment:
There is such feature https://www.rabbitmq.com/queues.html#optional-arguments

It's possible to set some signature when worker creates a queue. If queue already exists worker can analyze queue signature. If signature is different that seems to be a collision of two services.

Signature may be name of project or something else which is permanet for a service.

Reply:
It's interesting but how to distinguish same queue for another project from same queue declared by same project previously? If they have different names - conflict is already solved, if same names - no way to distinguish.

Reply:
I guess the situation with two same queues in one project is incorrect by design. Agree? We can detect it and raise an exception while a worker is starting.

Reply:
The problem is that worker starts, successfully creates it's queue, then shuts down and __never__ starts again because this queue already exists. How to handle this?

Reply:
I think that queues name collision is not a reason to break a worker start. But it's a way to warn about a potential problem. We can send a message to log for example.

Reply:
In this case you'll have 100% false-positive warning rate, because each connect will cause this warning to happen.

Reply:
In any case, it's up to you!